AEPServices/Sources/ui/fullscreen/FullscreenMessage.swift

```diff
@@ -51,12 +51,27 @@ public class FullscreenMessage: NSObject, WKNavigationDelegate, FullscreenPresen
         self.messageMonitor = messageMonitor
     }
 
+    public func hide() {
+        DispatchQueue.main.async {
+            if self.messageMonitor.dismiss() ==  false {
+                return
+            }
+            self.dismissWithAnimation(animate: true, shouldDeallocateWebView: false)
+        }
+    }
+
```

Comment:
In this function it is mandatory to call the `messageMonitor` to dismiss. So forth other InAppMessages can be shown when the current one is hidden

Reply:
If this is not defined in a protocol can you add doc comments for it?

Reply:
yeap that should be done